x-pack/legacy/plugins/canvas/i18n/units.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+
+export const UnitStrings = {
+  time: {
+    getSecondsText: (seconds: number) =>
+      i18n.translate('xpack.canvas.units.time.seconds', {
+        defaultMessage: '{seconds} {seconds, plural, one {second} other {seconds}}',
```

Comment:
You can write this instead

```
{seconds, plural, one {# second} other {# seconds}
```

the # here will return the value of `seconds`;